Makefile

```diff
@@ -288,6 +295,11 @@ OPERATOR_SDK = $(shell which operator-sdk)
 endif
 endif
 
+.PHONY: api-docs
+api-docs: crd-ref-docs ## Creates API docs using https://github.com/elastic/crd-ref-docs
+	$(CRD_REF_DOCS) --source-path ./ --output-path ./docs/api-overview.md --renderer markdown --config ./crd-ref-docs.config.yaml && \
```

Comment:
Are the && really necessary? They hurt readability quite a bit. :)

Reply:
Not really necessary, setting these up so we don't have to manually keep track of api docs as we update / expand our apis.

If there are any other tools, I am totally okay with integrating those